nms/app/context/CbsdContext.tsx

```diff
@@ -160,6 +161,11 @@ export async function remove(params: {networkId: string; cbsdId: number}) {
   });
 }
 
+interface PaginationOptions {
```

Comment:
We do not use Interfaces anywhere in the code. If it is a type definition, use of `type` is preferred.

Reply:
Changed to `type`.